app/assets/stylesheets/tracks.scss

```diff
@@ -1,3 +1,35 @@
 // Place all the styles related to the tracks controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.tracks-index {
```

Comment:
Would it be worth bemify these styles to keep it consistent with our newer components?

Reply:
probably

Reply:
I won't let it block this, but I think we should make it a requirement going forward.